app/models/fika.rb

```diff
@@ -3,6 +3,14 @@ class Fika < ApplicationRecord
   has_and_belongs_to_many :participants, join_table: 'fikas_participants', presence: true
   after_validation :create_calendar_entry
 
+  def formated_start_date
+    start_date.strftime('%-d/%-m %H:%M')    
+  end
+
+  def formated_end_date
+    end_date.strftime('%-d/%-m %H:%M')    
```

Comment:
We dont really use the end date anywhere on the front end, could it be removed? Or does G calendars need it?

Reply:
This can totally be removed! We've removed it.